src/components/checkbox/index.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { motion } from 'framer-motion';
+
+const Checkbox = () => {
+  const [clicked, setClicked] = useState(false);
+  return (
+    <CheckboxContainer
+      onClick={() => {
+        setClicked(!clicked);
+      }}
+    >
+      <HiddenCheckbox />
+      <StyledCheckbox clicked={clicked}>
+        {clicked ? (
+          <svg
```

Comment:
Just a matter of preference, but you could use this (maybe w a styled component instead of inline) instead of the svg:
```HTML
<div style={{color: 'white', fontSize: '1.35rem'}}>&#x2713;</div>
```
https://www.toptal.com/designers/htmlarrows/ is a nice reference for this stuff

Reply:
I chose to use a svg because it matched what was in the figma file the best. In fact, I just exported the checkmark svg from the figma file, so I think it should be fine.